client/src/plus/notifications/tabs.tsx

```diff
@@ -1,15 +1,20 @@
 import { useState, useEffect } from "react";
 import { useLocation } from "react-router-dom";
+import { COLLECTION_BASE_PATH } from "./api";
 
 export enum TabVariant {
-  ALL,
+  NOTIFICATIONS,
   STARRED,
   WATCHING,
+  COLLECTIONS,
+  FREQUENTLY_VIEWED,
 }
 
 const ALL_URL = "/plus/notifications";
 const STARRED_URL = "/plus/notifications/starred";
 const WATCHING_URL = "/plus/notifications/watching";
+const COLLECTIONS_URL = "/plus/collection";
+const FREQUENTLY_VIEWEED_URL = "/plus/collection/frequently_viewed";
```

Comment:
Should these be in a more central `constants.tsx`, possibly properties of a `PlusRoutes` or `PlusPaths` object?

Reply:
I have refactored this to make the tabs.tsx common for both consumers. I think in that context these constants can live there.